kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -124,6 +125,75 @@ public EntrySet getNode(Symbol sym, String signature) {
     return node;
   }
 
+  private static NodeBuilder setFlagProperties(Symbol sym, NodeBuilder builder) {
+    long flags = sym.flags();
+    switch(sym.getKind()) {
+      case ENUM:
+      case CLASS:
+      case ANNOTATION_TYPE:
+      case INTERFACE:
+        builder = builder.setProperty("access", accessFlag(flags));
```

Comment:
"Property" here is the same as "fact" in the Kythe schema, right? So this will end up creating entries with fact name "/kythe/access"?

I wonder if they will expect you to update the schema doc (`kythe/docs/schema/schema.txt`) appropriately? 

In fact, this adds a large number of new fact names - it more than doubles the current number. That may be fine - these are freeform strings and not enumerated values for a reason. I'm just trying to anticipate their pushback.  It may be worth pointing this out to them and checking that they don't have some reason for keeping the number of distinct fact names small.